src/entities/observation-entity.ts

```diff
@@ -53,8 +55,57 @@ export enum Verified {
   Rejected = 'rejected',
 }
 
+interface RawObservationBase<TCommon, TRing, TSpecies> {
+  ring: TRing;
```

Comment:
please add short description of purpose of this interfaces in comment, as it takes time to got

Reply:
+